packages/app/universal-ts-utils/src/array/sort.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Sorts an array of primitive types in either ascending or descending order.
+ *
+ * @template T The type of the elements in the array.
+ *
+ * @param {T[]} array - The array to be sorted. This should be an array of primitive values.
+ * @param {'asc' | 'desc'} [order='asc'] - The order in which to sort the array. Defaults to 'asc' if not specified.
+ *
+ * @returns {T[]} The sorted array in the specified order.
+ */
+export const sort = <T extends string | number | boolean>(
```

Comment:
I am not really sure, to be honest, I am adding this method because we are using them, as far as I know, on tests and we don't have a big amount of data there so maybe we can postpone that perf improvement but not sure if you have in mind another cases

Reply:
ok, makes sense then!